### DISCUSSION TITLE
6-2 part 7 how to fix ?

### DISCUSSION THREAD

**Discussion by khartman-nj** [ua-gist604b-f24/syllabus#36] [General]
Can anyone help me I am confused on how to do this step I accessed the server and I am trying to repair the data but I am not sure what is wrong with this dataset?

**Comment by travisz09** [comment]
Have you been able to get this far? Click on the second osm link under "Store Name".

![image](https://github.com/user-attachments/assets/f45c687b-fdf0-4b95-bc11-d63dc93ce284)

**Reply by khartman-nj** [reply]
I was but my codespace just lost connection with the server

**Reply by travisz09** [reply]
That's a bummer! Try to restart your codespace and see if that fixes it. If not you might just have to try again later.

**Reply by khartman-nj** [reply]
Got to that point is this where we enter the local host information?

**Reply by travisz09** [reply]
Yes. Update the Connection Parameters like so.

![image](https://github.com/user-attachments/assets/db9b005c-7627-4a05-889b-73cfb0ed5f40)

**Reply by khartman-nj** [reply]
![Error](https://github.com/user-attachments/assets/4be846d0-fba3-45bf-a865-09d1f995b58d)
I got this issue

**Reply by travisz09** [reply]
For additional explanation as to why this works see Professor Olsson's comments on [https://github.com/ua-gist604b-f24/syllabus/discussions/31](https://github.com/ua-gist604b-f24/syllabus/discussions/31), as well as the postgres database structure in your codespace.

![image](https://github.com/user-attachments/assets/b10d916d-3706-4c8c-8bc0-8cf8058b1314)

**Reply by travisz09** [reply]
Hmmm... Can you confirm that the database is running in codespace?

**Reply by travisz09** [reply]
I can recreate this exact error by stopping the postgres docker container. Make sure both containers are running. If that still doesn't work, I would suggest trying again later.

![image](https://github.com/user-attachments/assets/9a45bd48-105e-4658-8f4a-ba0bc115d855)

**Reply by khartman-nj** [reply]
It is not connected in the codespace trying to figure out why this is the case

**Reply by travisz09** [reply]
I would try starting over at section 4. Then access the gdb through the PostgreSQL extension using the method detailed in 5-1.

**Reply by khartman-nj** [reply]
I will give it a shot

**Reply by khartman-nj** [reply]
So I restarted the code space again I go to docker everything is blank and I enter the password but no Hawaii
![error2](https://github.com/user-attachments/assets/3078553c-7974-4bce-90fc-6cc1f0f1899d)
 info comes up any suggestions

**Reply by travisz09** [reply]
Use the host name 'localhost' in codespace, not 'postgres'. It is terribly confusing.

**Reply by travisz09** [reply]
In the "Add Database Connection" prompt, the sequence should be:
1. localhost
2. postgres
3. postgres
4. 5432
5. Standard Connection
6. hawaii
7. hawaii (or any other name you choose)

**Reply by khartman-nj** [reply]
That worked and then are we suppose to open the package and take a screenshot in QGIS I am confused on what deliverable our teacher wants us to produce. Sorry this lesson has thrown me for a loop.

**Reply by travisz09** [reply]
That's OK. I think Professor Olsson is working on a tutorial for this, but my understanding is that you just need to open the layer group as a preview in geoserver and take a screenshot zoomed into Hawaii. Something like this.

![image](https://github.com/user-attachments/assets/6f5ec4bf-7d5a-4e30-a060-2476111e51b9)

The image should be initially a global scale, but the detailed layers (being served by our postgres gdb) are only available for Hawaii. Zoom in to see the details.

**Reply by aaryno** [reply]
I think one way to fix this, if you want to have this work more cleanly for an individual layer, is to edit the layer and re-calculate the layer extents. Then when you open the Layer Preview it should be zoomed into the area where there's actually data. Probably by default the layer extents are -180, -90, 90, 180

**Reply by khartman-nj** [reply]
Thankyou Travis and Dr.Olsson for your help on this. Issue resolved
